src/raw_logging.cc

```diff
@@ -120,16 +109,13 @@ void RawLog__(LogSeverity severity, const char* file, int line,
     return;  // this stderr log message is suppressed
   }
   // can't call localtime_r here: it can allocate
-  struct ::tm& t = last_tm_time_for_raw_log;
   char buffer[kLogBufSize];
   char* buf = buffer;
   int size = sizeof(buffer);
 
   // NOTE: this format should match the specification in base/logging.h
```

Comment:
do we still need to match the format?

abseil's internal raw_logging doesn't include timestamp.
https://github.com/abseil/abseil-cpp/blob/master/absl/base/internal/raw_logging.cc#L153

Reply:
I just followed the NOTE comment, but I would also think that it is not necessary to keep the format the same.

Btw. I don't know what happened with appveyor VS builds (mingw was ok). I'd have some trouble to get a windows/VS environment and try to reproduce.